src/browser/vdiff.js

```diff
@@ -1,11 +1,20 @@
 import { chai, expect } from '@open-wc/testing';
 import { executeServerCommand } from '@web/test-runner-commands';
 
-chai.Assertion.addMethod('golden', ScreenshotAndCompare);
+mocha.setup({ // eslint-disable-line no-undef
+	rootHooks: {
+		beforeEach() {
+			const { currentTest } = this;
+			chai.Assertion.addMethod('golden', function(...args) {
+				return ScreenshotAndCompare.call({ test: currentTest, elem: this._obj }, ...args); // eslint-disable-line no-invalid-this
```

Comment:
I guess `this._obj` is a Chai thing eh, like it's whatever you call `expect(thing).to.be.whatever` with?